css/doc_site_styling.scss

```diff
@@ -245,6 +245,8 @@ figure {
 
 // UTIL Specific Tables
 .utils-table {
+  margin-bottom: var(--s-space-48);
+
```

Comment:
Instead of adding margin to the tables, can we instead add the margin utility class to headers where needed in the `utils.html`?

E.g.

```html
<h2 id="text-align" class="s-h2 s-mt-5">Text Align</h2>
```

Reply:
Yes, YAY! I forget we have them now 🤣